src/routes/RequestsRoute.js

```diff
@@ -246,6 +262,8 @@ export const getListFormatter = (
   'year': rq => getFormattedYears(rq.instance?.publication, DEFAULT_DISPLAYED_YEARS_AMOUNT),
   'callNumber': rq => effectiveCallNumber(rq.item),
   'servicePoint': rq => get(rq, 'pickupServicePoint.name', DEFAULT_FORMATTER_VALUE),
+  'copies': rq => get(rq, 'printDetails.count', DEFAULT_FORMATTER_VALUE),
```

Comment:
From a consistency perspective and best practices, keeping all access key strings constant in separate files makes sense.